app/views/audio-video/reservations.js

```diff
@@ -581,12 +581,11 @@ define(['app', 'lodash', 'moment', 'jquery',
                     })
                 }
 
-                if(!doc.youtubeKeysLoaded && _.intersection(user.roles, ['Administrator', 'EunoAdministrator']).length > 0 && doc.interactioEventId){
```

Comment:
Maybe the user role check is better else it will query for any users unnecessarily